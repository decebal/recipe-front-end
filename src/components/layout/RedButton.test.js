import React from 'react';
import renderer from 'react-test-renderer';
import RedButton from './RedButton';


test('RedButton renders correctly', () => {
  const tree = renderer
    .create(<RedButton />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
