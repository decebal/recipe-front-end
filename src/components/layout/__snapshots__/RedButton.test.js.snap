// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedButton renders correctly 1`] = `
<button
  className="sc-bdVaJa eOLpdL"
/>
`;
